### DISCUSSION TITLE
Great progress!

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#10] [General]
@brianzelip 

The site is looking dope! Is there anything I can help with? I see the pages are developing more! Lmk what I can do.

**Comment by brianzelip** [comment]
@j-p-courneya  You can outline or flesh out the membership page with content — words related to joining and renewing. You can put it here or in some markdown file in the admin folder.

Next I’m going to start looking into pulling in google data for the news and events page.

**Reply by j-p-courneya** [reply]
@brianzelip 

I think the relay Facebook content is most relevant as our "news" I send out email blasts and had imagined that content would be what ends up as news... I will put stuff on the calendar too 

I will work on the ria membership info

**Reply by brianzelip** [reply]
@j-p-courneya I don’t look at FB so I don’t know what that info looks like. If you add some examples to the repo, either here in discussions or in markdown, that would be helpful for me.

**Reply by j-p-courneya** [reply]
@brianzelip 
This sounds like a good opportunity to understand how to post news here. One thing I worry about is cross platform posting fatigue. My current workflow is. I write an email, then I post the same content in BAND a messaging platform we use. Then the person who manages Facebook is doing their own postings, it's unwieldy!

It would be cool if there were a way to streamline that if you have any ideas. Currently the only thing upcoming is the covered Dish dinner on March 17th at the town hall.

**Comment by brianzelip** [comment]
@j-p-courneya outline or flesh out Town Hall content too.

**Comment by brianzelip** [comment]
Done
